pkg/sdk/users.go

```diff
@@ -426,8 +432,14 @@ type UserUnset struct {
 
 func (opts *UserUnset) validate() error {
 	// TODO [SNOW-1645875]: change validations with policies
```

Comment:
this TODO is still valid after these changes?

Reply:
It is; it's about adding support of { auth | password | session } policies on user resource.

Reply:
But you have already added the validations for them (and the TODO is about changing the validations with policies). Is something moar needed here?

Reply:
From the ticket description, adding them to the user schema (?)

Reply:
Comment was left, but it seems that there are no more validations needed. Will be removed.